From 80dd397ce4ff7d0aa6b81fab5cc03a10e6e3b406 Mon Sep 17 00:00:00 2001
From: nikolas <nikolas@gnu.org>
Date: Thu, 7 Mar 2019 12:52:57 -0500
Subject: [PATCH 0460/1053] Fix: spelling for a few real town names (#7338)

Added special characters to mostly Slovak towns, a few Spanish, and one Turkish.
---
 src/table/townname.h | 86 ++++++++++++++++++++++----------------------
 1 file changed, 43 insertions(+), 43 deletions(-)

diff --git a/src/table/townname.h b/src/table/townname.h
index 057583177d..551a050697 100644
--- a/src/table/townname.h
+++ b/src/table/townname.h
@@ -645,7 +645,7 @@ static const char * const _name_spanish_real[] = {
 	"San Carlos",
 	"El Lim\xC3\xB3n",
 	"Coro",
-	"Corocoro",
+	"Coro Coro",
 	"Puerto Ayacucho",
 	"Elorza",
 	"Arismendi",
@@ -668,7 +668,7 @@ static const char * const _name_spanish_real[] = {
 	"Ibarra",
 	"San Lorenzo",
 	"Macas",
-	"Morana",
+	"Mora\xC3\xB1""a",
 	"Machala",
 	"Zamora",
 	"Latacunga",
@@ -2036,28 +2036,28 @@ static const char * const _name_romanian_real[] = {
 
 static const char * const _name_slovak_real[] = {
 	"Bratislava",
-	"Banovce nad Bebravou",
+	"B\xC3\xA1novce nad Bebravou",
 	"Bansk\xC3\xA1 Bystrica",
 	"Bansk\xC3\xA1 \xC5\xA0tiavnica",
 	"Bardejov",
 	"Brezno",
 	"Brezova pod Bradlom",
-	"Bytca",
-	"Cadca",
-	"Cierna nad Tisou",
+	"Byt\xC4\x8D""a",
+	"\xC4\x8C""adca",
+	"\xC4\x8Cierna nad Tisou",
 	"Detva",
 	"Pre\xC5\xA1ov",
 	"Dolny Kubin",
 	"Spi\xC5\xA1sk\xC3\xA1 Nov\xC3\xA1 Ves",
-	"Dunajska Streda",
-	"Gabcikovo",
+	"Dunajsk\xC3\xA1 Streda",
+	"Gab\xC4\x8D\xC3\xADikovo",
 	"Galanta",
 	"Gbely",
 	"Gelnica",
-	"Handlova",
+	"Handlov\xC3\xA1",
 	"Hlohovec",
-	"Holic",
-	"Humenne",
+	"Hol\xC3\xAD\xC4\x8D",
+	"Humenn\xC3\xA9",
 	"Hurbanovo",
 	"Kezmarok",
 	"Kom\xC3\xA1rno",
@@ -2066,61 +2066,61 @@ static const char * const _name_slovak_real[] = {
 	"Krompachy",
 	"Kuty",
 	"Leopoldov",
-	"Levoca",
-	"Liptovsky Mikulas",
-	"Lucenec",
+	"Levo\xC4\x8D""a",
+	"Liptovsk\xC3\xBD Mikul\xC3\xA1\xC5\xA1",
+	"Lu\xC4\x8D""enec",
 	"Malacky",
 	"Martin",
 	"Medzilaborce",
 	"Michalovce",
 	"Modra",
 	"Myjava",
-	"Namestovo",
+	"N\xC3\xA1mestovo",
 	"Nitra",
 	"Nov\xC3\xA1 Ba\xC5\x88""a",
 	"Nov\xC3\xA9"" Mesto nad V\xC3\xA1hom",
 	"Nov\xC3\xA9"" Z\xC3\xA1""mky",
-	"Partizanske",
+	"Partiz\xC3\xA1nske",
 	"Pezinok",
-	"Piestany",
-	"Poltar",
+	"Pie\xC5\xA1\xC5\xA5""any",
+	"Polt\xC3\xA1r",
 	"Poprad",
 	"Pova\xC5\xBEsk\xC3\xA1 Bystrica",
 	"Prievidza",
-	"Puchov",
-	"Revuca",
-	"Rimavska Sobota",
-	"Roznava",
-	"Ruzomberok",
+	"P\xC3\xBA""chov",
+	"Rev\xC3\xBA""ca",
+	"Rimavsk\xC3\xA1 Sobota",
+	"Ro\xC5\xBE\xC5\x88""ava",
+	"Ru\xC5\xBEomberok",
 	"Sabinov",
-	"Sala",
+	"\xC5\xA0""a\xC4\xBE""a",
 	"Senec",
 	"Senica",
-	"Sered",
+	"Sere\xC4\x8F",
 	"Skalica",
-	"Sladkovicovo",
+	"Sl\xC3\xA1""dkovi\xC3\x8Dovo",
 	"Smolenice",
 	"Snina",
-	"Stara Lubovna",
-	"Stara Tura",
-	"Strazske",
+	"Star\xC3\xA1 \xC4\xBDubov\xC5\x88""a",
+	"Star\xC3\xA1 Tur\xC3\xA1",
+	"Str\xC3\xA1\xC5\xBEske",
 	"Stropkov",
 	"Stupava",
-	"Sturovo",
-	"Sulekovo",
-	"Topolcany",
-	"Trebisov",
-	"Trencin",
+	"\xC5\xA0t\xC3\xBArovo",
+	"\xC4\xA0ulekovo",
+	"Topo\xC4\xBE\xC4\x8D""any",
+	"Trebi\xC5\xA1ov",
+	"Tren\xC4\x8D\xC3\xADn",
 	"Trnava",
-	"Turcianske Teplice",
-	"Tvrdosin",
-	"Vrable",
-	"Vranov nad Toplou",
-	"Zahorska Bystrica",
-	"Zdiar",
-	"Ziar nad Hronom",
+	"Tur\xC4\x8Dianske Teplice",
+	"Tvrdo\xC5\xA1\xC3\xADn",
+	"Vr\xC3\xA1""ble",
+	"Vranov nad Top\xC4\xBEou",
+	"Z\xC3\xA1horsk\xC3\xA1 Bystrica",
+	"\xC5\xBD""diar",
+	"\xC4\xBDiar nad Hronom",
 	"\xC5\xBDilina",
-	"Zlate Moravce",
+	"Zlat\xC3\xA9 Moravce",
 	"Zvolen",
 };
 
@@ -2834,7 +2834,7 @@ static const char * const _name_turkish_real[] = {
 	"Edirne",
 	"Elaz\xC4\xB1\xC4\x9F",
 	"Erzurum",
-	"Eskisehir",
+	"Eski\xC5\x9F""ehir",
 	"Giresun",
 	"G\xC3\xBCm\xC3\xBC\xC5\x9Fhane",
 	"Hatay",
-- 
2.55.0

